common/src/main/java/com/wynntils/features/HealthPotionBlockerFeature.java

```diff
@@ -30,8 +30,8 @@ public void onPotionUse(PacketSentEvent<ServerboundUseItemPacket> e) {
 
         ItemStack stack = McUtils.inventory().getSelected();
 
-        // TODO check for crafting potion
-        if (!WynnItemMatchers.isHealingPotion(stack)) return;
+        if (!WynnItemMatchers.isHealingPotion(stack)
+                && !WynnItemMatchers.isCraftedHealingPotion(stack)) return;
```

Comment:
I think that these 2 methods could be potentially merged into one

Reply:
This would also remove the duplicated checks in the methods

Reply:
> I think that these 2 methods could be potentially merged into one

Thought so too, but maybe there would be some use cases where it could be useful to have them separate? Also, the amount of duplicated checks is really low.
Let's get someone else's option this and I'll refactor accordingly.

Reply:
Question: Does isHealingPotion() return true or false for crafted healing potions? If it's the former, then the test will never get to the isCraftedHealingPotion() test. If it's the latter, it definitely feels like an odd design -- why would not a crafted healing potion be a healing potion?

Reply:
`isHealingPotion` does not return true for crafted pots because names are randomized.

Reply:
If I were to merge the 2 functions, should I just call `isCraftedHealingPotion` in `isHealingPotion` or just merge them into a single function?